src/main.rs

```diff
@@ -58,7 +65,7 @@ fn main() {
         // we use both IO and Time tokio utilities
         .enable_all()
         // give our threads a descriptive name (they'll be numbered too)
-        .thread_name("acorn-tokio-thread")
+        .thread_name("hc-runner-tokio-thread")
```

Comment:
```suggestion
        .thread_name("holochain-runner-tokio-thread")
```

good point I think maybe we should go full title match tho